.gitignore

```diff
@@ -9,4 +9,4 @@
 *.lk
 *.int
 *.backup
-*.int
```

Comment:
you have a commit on your branch that isn't on master that changes gitignore.

You should probably do a fresh checkout of upstream/master, and cherry-pick only the changes

Unfortunately you can't .gitignore the tgstation.int as it's already comitted, instead use this command

git update-index --assume-unchanged tgstation.int to ignore forever more all local changes to tgstation.int